forwards_paper.tex

```diff
@@ -196,14 +196,40 @@ \section*{Results}
 %%%%%%
 \subsection*{Simulation benchmarks}
 
-\plr{Jaime and Kevin to describe simulation methods here (separately).}
-
-Comparison of simulation with/without msprime, using \simupop{}
-or maybe just a simple haploid simulation with 1000 QTL and stabilizing selection on a trait (say).
-
-Maybe an estimate of how long \emph{just} the ARG recording and simplification takes,
-so that then we can say how fast the simulator would have to be to do $10^6$ whole chromosomes for $10^7$ generations
-in a day.
+To measure the performance gains from ancestry recording, we ran simulations implemented in C++ (see above) with and
+without ancestry tracking.  The simulations without ancestry tracking introduced neutral mutations at a scaled mutation
```

Comment:
Unless we add neural stuff from simupop (which I think we should do, see below), the above really only describes the fwdpy stuff. Regardless of whether we do that, think it'd be nice to introduce the overall parameters for both before diving into simupop or fwdpy specific bits